src/rest-server/src/models/v2/virtual-cluster/k8s.js

```diff
@@ -81,12 +83,28 @@ const getNodeResource = async () => {
   const pods = await getPodsInfo();
   const nodeResource = {};
 
-  for (let node of Object.keys(vcData.clusterNodeGpu)) {
-    nodeResource[node] = {
-      gpuTotal: vcData.clusterNodeGpu[node].gpu,
-      gpuUsed: 0,
-      gpuAvaiable: vcData.clusterNodeGpu[node].gpu,
-    };
+  if (launcherConfig.enabledHived) {
+    for (let node of Object.keys(vcData.clusterNodeGpu)) {
+      nodeResource[node] = {
+        gpuTotal: vcData.clusterNodeGpu[node].gpu,
+        gpuUsed: 0,
+        gpuAvaiable: vcData.clusterNodeGpu[node].gpu,
+      };
+    }
+  } else {
+    const nodes = (await axios({
+      method: 'get',
+      url: `${kubernetesConfig.apiserver.uri}/api/v1/nodes`,
+    })).data.items.filter((node) => node.metadata.labels['pai-worker'] === 'true');
```

Comment:
Will this be called only once at init or every time refresh the VC page?

Reply:
every time when api is called, there may be node addition/deletion so getting the results at the beginning may not consistent

Reply:
getting used resources or running pods also call apiserver every time when the same api is called in previous implementation, so I assume it's ok for apiserver?

Reply:
ok, current is fine. we would better to watch and cache it in future.